src/Symfony/Bridge/Twig/Attribute/Template.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Attribute;
+
+#[\Attribute(\Attribute::TARGET_METHOD | \Attribute::TARGET_FUNCTION)]
+class Template
+{
+    public function __construct(
+        public string $template,
+        public ?array $vars = null,
```

Comment:
```suggestion

        /**
         * The controller method arguments to pass to the template.
         */
        public ?array $vars = null,
```

Assumes I'm correct in my understanding of `$vars`.

Reply:
That's it. Example from [SensioFrameworkExtraBundle doc]:(https://symfony.com/bundles/SensioFrameworkExtraBundle/current/annotations/view.html)
```php
#[ParamConverter('post', class: 'SensioBlogBundle:Post')]
#[Template('@SensioBlog/post/show.html.twig"', vars: ['post'])]
public function show(Post $post) {}
```

Reply:
updated